include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -1100,6 +1134,185 @@ namespace internal
           }
       }
 
+
+
+      /**
+       * This evaluates the mapping information on a range of cells calling
+       * into the tensor product interpolators of the matrix-free framework,
+       * using a polynomial expansion of the cell geometry in terms of
+       * MappingQ.
+       */
+      template <int dim, typename Number, typename VectorizedArrayType>
+      void
+      compute_range_mapping_q(
+        const unsigned int               begin_cell,
+        const unsigned int               end_cell,
```

Comment:
Cell or macro cell?

Reply:
These are cell batches, i.e., macro cells. However, we use the terminology `cell` for both single cells and cell batches throughout that file already, and I think it would only lead to confusion if we changed the name in a single spot here. I think the meaning is clear from the context.